app/views/admin/calendars/_form.html.erb

```diff
@@ -21,7 +21,7 @@
                           hint: 'Which group organises these events?' %>
 
         <%= f.input :name, hint: 'A simple description to help rememember what this calendar is' %>
-        <%= f.input :source, label: 'URL', placeholder: 'https://your-domain.com/events.ics', hint: "The source URL for your calendar feed. See the #{link_to 'PlaceCal Handbook', 'https://handbook.placecal.org/admin-handbook'} if you need help.".html_safe %>
+        <%= f.input :source, label: 'URL', placeholder: 'https://your-domain.com/events.ics', hint: t('admin.calendars.handbook_hint_html').html_safe %>
```

Comment:
I don't think you need the `.html_safe` here anymore but I may be misunderstanding the magic powers of the locale key slug `_html`

Reply:
oh i see!! i didn't know that! is this a normal ish thing for using locale files? thanks for the tip :)

Reply:
I've not encountered before. Seems like a special rails thing. I _think_ it makes sense though to declare the html escape in the place where you add it.